portfolio/src/main/webapp/Kite_gallery.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html id = "gallery">
```

Comment:
minor comment: the whitespace around `=` is not consistent across the file.

https://google.github.io/styleguide/htmlcssguide.html#HTML is the styleguide.